x/staking/keeper/val_state_change.go

```diff
@@ -36,6 +36,7 @@ func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []ab
 
 	// Iterate over validators, highest power to lowest.
 	iterator := sdk.KVStoreReversePrefixIterator(store, ValidatorsByPowerIndexKey)
+	defer iterator.Close()
 	count := 0
```

Comment:
The iterator will be created in every endBlocker. This missing may cause huge memory leak.

Reply:
yikes
\